src/wwwroot/controllers/MainCtrl.js

```diff
@@ -189,6 +190,9 @@
     $rootScope.$on('$locationChangeStart', function () {
       submenues = [];
     });
+    $rootScope.getLoadedLimits = function () {
+        $rootScope.$broadcast('LimitsLoaded');
```

Comment:
I think that we do not need it, but, if we really need it could be done in the same place where _$rootScope.accountLimits_ is updated without using _watch_